src/Serilog.Sinks.Elasticsearch/Serilog.Sinks.Elasticsearch.csproj

```diff
@@ -50,6 +50,10 @@
       <HintPath>..\..\packages\Elasticsearch.Net.2.0.4\lib\net45\Elasticsearch.Net.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=8.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
It's an error that this was included. Fixed in 87ba8e4d3dc10c0e5c6b6ac0bc43d81c5c2cda63